src/Models/AzureFunctions.ts

```diff
@@ -48,7 +42,8 @@ export class AzureFunctions implements Component, Provisionable, Deployable {
   private azureAccountExtension: AzureAccount | undefined = getExtension(ExtensionName.AzureAccount);
   private functionLanguage: string | null;
   private functionFolder: string;
-
+  private projectRootPath: string;
+  private azureConfigFileHandler: AzureConfigFileHandler;

```

Comment:
I am not sure if we could make AzureConfigFileHandler as a utility instead of always passing it to consturctor?

Reply:
I ever considered this way, this is likely to make "AzureConfigFileHandler" as a static class. I was not so sure if all our projects would need Azure Config support. So I just leave it as it is.